### PR TITLE
EXTPLESK-4816 Disabled not recommended way to authorize

### PR DESCRIPTION
**Pull request by AlinaADmi**

### CODE REVIEW

**Review by vbaranovskiy-plesk — APPROVED** [review]

**Review by maksimsavrilov — APPROVED** [review]

File: src/plib/resources/locales/en-US.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
     'authDataSaved' => 'The authorization data was saved.',
     'getAuth' => 'Find your root account credentials here',
     'getAuthStepTwo' => 'You can also create one yourself. If you do, make sure to grant the "route53:*" and "route53domains:*" permissions to the account (<a href="%%learnMoreUrl%%">learn more</a>).',
+    'getAuthNoRoot' => 'Please, make sure to grant the "route53:*" and "route53domains:*" permissions to the account (<a href="%%learnMoreUrl%%">learn more</a>).',
```
Comment (GolovizninVV) [inline src/plib/resources/locales/en-US.php]: Does it refer to the AWS account?

File: src/plib/resources/locales/en-US.php [hunk 2]
```diff
@@ -42,7 +43,8 @@
     'syncAllDone' => 'All zones registered in Plesk were synchronized with Amazon.',
     'removeAllDone' => 'All zones hosted on Amazon were removed.',
     'awsApiAuthorization' => 'AWS API Authorization',
-    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon. If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
+    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon.',
+    'awsDescriptionWindowRoot' =>' If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
```
Comment (GolovizninVV) [inline src/plib/resources/locales/en-US.php]: Do "root account credentials" refer to Plesk or to Amazon?

File: src/plib/resources/locales/en-US.php [hunk 3]
```diff
@@ -42,7 +43,8 @@
     'syncAllDone' => 'All zones registered in Plesk were synchronized with Amazon.',
     'removeAllDone' => 'All zones hosted on Amazon were removed.',
     'awsApiAuthorization' => 'AWS API Authorization',
-    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon. If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
+    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon.',
```
Comment (GolovizninVV) [inline src/plib/resources/locales/en-US.php]: * What does "restricted IAM user account" mean? I can't find such term for Amazon.
* Does it all mean that one needs to authenticate in AWS using such account before managing Route 53?

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# 2.9.1
+
+* [+] Integration with AWS root account is not available anymore because of security and recommendation to avoid using this way.
```
Comment (GolovizninVV) [inline CHANGES.md]: * Is it a new feature? For me, it looks more like removing the non-supported feature.
* Does Amazon no longer provides authentication using root account credentials? If so, is it only for Plesk or for others as well?

File: src/plib/resources/locales/en-US.php (hunk 2 above)
Comment (AlinaADmi) [reply src/plib/resources/locales/en-US.php]: Root account refer to Amazon

File: src/plib/resources/locales/en-US.php (hunk 1 above)
Comment (AlinaADmi) [reply src/plib/resources/locales/en-US.php]: image
I think it should help

Comment (AlinaADmi) [reply src/plib/resources/locales/en-US.php]: So, it`s refer to AWS account

File: src/plib/resources/locales/en-US.php (hunk 3 above)
Comment (AlinaADmi) [reply src/plib/resources/locales/en-US.php]: Restricted user is a user with limited rights. 
Before fix there was an ability to login with root account creds  or use only 'limited' account. Now we allow to login only the second way of authentication.

https://docs.aws.amazon.com/signin/latest/userguide/console-sign-in-tutorials.html

File: CHANGES.md (hunk 4 above)
Comment (AlinaADmi) [reply CHANGES.md]: Amazon supports, but we removing such way of auth for security reasons

Comment (AlinaADmi) [reply CHANGES.md]: Amazon guys does not recommend such way of auth for regular tasks: https://docs.aws.amazon.com/signin/latest/userguide/introduction-to-root-user-sign-in-tutorial.html

Comment (GolovizninVV) [reply CHANGES.md]: [*] [For the sake of security](https://docs.aws.amazon.com/signin/latest/userguide/introduction-to-root-user-sign-in-tutorial.html), we removed the ability to sign in to Amazon Route 53 using the account's root user credentials.

File: src/plib/resources/locales/en-US.php (hunk 3 above)
Comment (GolovizninVV) [reply src/plib/resources/locales/en-US.php]: You need an IAM user to manage Amazon Route 53.

File: src/plib/resources/locales/en-US.php (hunk 2 above)
Comment (GolovizninVV) [reply src/plib/resources/locales/en-US.php]: If you provide credentials to your account root user, Plesk will create an IAM user in AWS for you.

File: src/plib/resources/locales/en-US.php (hunk 1 above)
Comment (GolovizninVV) [reply src/plib/resources/locales/en-US.php]: Make sure that <a href="%%learnMoreUrl%%">you have granted the "route53:*" and "route53domains:*" permissions</a>  to your IAM user.

**Pull request merged**